### DISCUSSION TITLE
Search in storage

### DISCUSSION THREAD

**Discussion by organicnz** [supabase/supabase#11920] [Feature Requests]
Hi folks,

Could you implement a search function in the storage? It would be very handy for managing media files through the Supabase website :)

Thank you very much :)

**Comment by logemann** [comment]
I would add that tagging would be nice. This is kind of similar to a search in a specific way. I am coming from cloudinary and there tagging is nicely implemented there. 

Lets say you have a Gallery with some pictures... you could tag those "gallery" or something like "customer123:gallery"

**Comment by devidevio** [comment]
It's already possible, check the docs:
* https://supabase.com/docs/reference/javascript/storage-from-list (hit the button named "Search files in a bucket").

```

const { data, error } = await supabase
  .storage
  .from('avatars')
  .list('folder', {
    limit: 100,
    offset: 0,
    sortBy: { column: 'name', order: 'asc' },
    search: 'jon' // <-- Search for "jon"
  })

```

**Reply by oddmutou** [reply]
whta is `'jon'` ?
what this query work any column of `storage.objects` table?
there is no sufficient explanation.
i think this issue is not "already possible".
